### ISSUE TITLE
man_made=reservoir_covered's OSM link is node, not way

### ISSUE THREAD

**Issue by goldfndr** [osmandapp/Osmand#5168]
What version of OsmAnd are you using, on what device and Android/iOS version?
Nightly build 2.9.0#24123m on rooted Moto G (2013, VZW XT1028) with KitKat 4.4.4

Do you use the offline maps offered within the OsmAnd app for download, or online (tile / raster) maps?
Offline

If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs, and its edition date.
Us_washington_northamerica_18_03_25.obf (OSM live)

When I tap on a recently mapped drain, the description for it links to https://www.openstreetmap.org/node/572775289 rather than the correct https://www.openstreetmap.org/way/572775289 link.

I examined it with `inspector`, and the tagging in the obf looks proper:
```xml
<way id = '4175' version='1'>
	<tag k='man_made' v='reservoir_covered' />
	<tag k='access' v='no' />
	<tag k='layer' v='-1' />
	<tag k='original_id' v='572775289'/>
	<tag k='osmand_id' v='73315237059'/>
	<nd ref = '4170'/>
	<nd ref = '4171'/>
	<nd ref = '4172'/>
	<nd ref = '4173'/>
	<nd ref = '4174'/>
</way>
```
Another way I checked, 344764242, is included in this file and doesn't have this issue, but it's not version 1.

**Comment by vshcherb** [comment]
Everything looks correct but I didn't manage to reproduce it :( I'm afraid I don't know how to tap on the object without name.

**Comment by goldfndr** [comment]
Oh, you would just tap on the "MM" (man_made) icon within the object.

**Comment by goldfndr** [comment]
Can this be reopened? If you're unable to tap on the object without name (which I can) then that'd be a separate bug.

**Comment by goldfndr** [comment]
For what it's worth, this is resolved as of recent nightly (3.5.6#28800m), probably resolved long ago.
